src/terms/hartree.jl

```diff
@@ -0,0 +1,24 @@
+struct Hartree end
```

Comment:
Coulomb is tricky because local, nonlocal, ewald, psp correction, hartree and xc all come from coulomb. I don't want to use potential because potential usually means a given field that acts as a multiplication operator: the hartree term gives a hartree energy, which itself gives a local (state-dependent) potential. I think Hartree is fairly standard in condensed matter

Reply:
Ok.